go-controller/pkg/util/ovn.go

```diff
@@ -75,3 +81,56 @@ func UpdateRouterSNAT(router string, externalIP net.IP, logicalSubnet *net.IPNet
 
 	return nil
 }
+
+// CreateMACBinding Creates MAC binding in OVN SBDB
+func CreateMACBinding(logicalPort, datapathName string, portMAC net.HardwareAddr, nextHop net.IP) error {
+	datapath, err := GetDatapathUUID(datapathName)
+	if err != nil {
+		return err
+	}
+
+	// Check if exact match already exists
+	stdout, stderr, err := RunOVNSbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "MAC_Binding",
+		"logical_port="+logicalPort,
+		fmt.Sprintf(`mac="%s"`, portMAC),
+		"datapath="+datapath,
+		fmt.Sprintf("ip=\"%s\"", nextHop))
+	if err != nil {
+		return fmt.Errorf("failed to check existence of MAC_Binding entry of (%s, %s, %s, %s)"+
+			"stderr: %q, error: %v", datapath, logicalPort, portMAC, nextHop, stderr, err)
+	}
+	if stdout != "" {
+		klog.Infof("The MAC_Binding entry of (%s, %s, %s, %s) exists with uuid %s",
+			datapath, logicalPort, portMAC, nextHop, stdout)
+		return nil
+	}
+
+	// Create new binding
+	_, stderr, err = RunOVNSbctl("create", "mac_binding", "datapath="+datapath, fmt.Sprintf("ip=\"%s\"", nextHop),
```

Comment:
Stylistically speaking we might need to define a proper style for these `RunOVN*` commands, the one above has a completely different line separation. Not something essential though, and I think a dedicated PR for that could be needed, so that our code base is changed all at once.

Reply:
we need to just go to go-ovn :)